tools/github-actions/release/action.yml

```diff
@@ -10,6 +10,10 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Previous release tag
+      if:  ${{!contains( inputs.version, '-' )) && endsWith( inputs.version, '0' )}}
+      shell: bash
+      run: echo "PREVIOUS_VERSION_TAG=$(gh release list --exclude-drafts --exclude-pre-releases --json tagName --limit 1 --jq '.[].tagName')" >> $GITHUB_ENV
```

Comment:
if I understand correctly, this will extract the tag of the latest release published on GitHub (chronologically)
however I fail to understand how this can be the previous tag to use for the generation of the notes regardless of the `inputs.version`

Reply:
Yes, discovered after testing that it was chronological order (I don't know wh I was thinking it was semver ordered).
I am working on a fix